uber/templates/registration/register.html

```diff
@@ -1,131 +1,64 @@
-<!DOCTYPE HTML>
```

Comment:
Oooh, good point. Though I was kind of thinking that these removals would be implemented inside the plugin, that doesn't solve the first point. We would have to also specify the inclusion page as a filter.

I will probably do that when I refactor these into two pages, so it fits better with the way plugins work - long-term, I would prefer not to have to remember specific removal functions to override. :)